spec/folio_client/authenticator_spec.rb

```diff
@@ -73,6 +73,15 @@
       end
     end
 
+    context "when the service returns a 409 conflict error" do
+      let(:http_status) { 409 }
+      let(:http_body) { "{\"error\" : \"get bent\"}" }
```

Comment:
Folio is so rude!

Reply:
Why should it be more polite with this error than any other? 😆

Reply:
At least it's consistent 🕺🏻